### DISCUSSION TITLE
Location mode.  Enabled in configuration but not seeing entities for it once enabled

### DISCUSSION THREAD

**Discussion by lambros414** [tsightler/ring-mqtt#688] [General Discussion]
I turned this feature on in the configuration screen:  

enable_modes   ( Enable support for Location Modes for sites without a Ring Alarm Panel )

Restarted the add-on and also restarted home assistant.  Despite this, I am not seeing <location modes> in the entity sections for my cameras.

I don't have a ring alarm panel, but it appears that this is what that option is for.

Is this a bug, have I done something wrong that I need to do differently?  

I have my own alarm system and panel within home assistant and would like to switch the Location mode <Away, Home, Disable> for my ring cameras based upon the status of my home assistant alarm system

**Comment by lambros414** [comment]
A follow up.  I noticed there is a similar thread on a discussion (Home mode no longer available #423).  Seems to be the same issue I am experiencing, but in that thread I have not seen a final solution

**Comment by tsightler** [comment]
Unfortunately, it's impossible to know if it's a bug or if something is configured incorrectly just based on a description.  Feel free to create an issue and provide full logs which will show the discovery messages and allow proper troubleshooting.

**Reply by lambros414** [reply]
I will feel free to create an issue and give logs, etc, however, a question that may answer everything, however I just want to confirm it isn't user error or misunderstanding vs a bug/config issue.....

per the instructions, I enabled
<img width="1058" alt="Screenshot 2023-07-27 at 12 41 20 PM" src="https://github.com/tsightler/ring-mqtt/assets/57337383/6701b80c-ebc4-485d-8301-b66d2539a7b4">

I then rebooted home assistant entirely (just to make sure everything is loaded.  then look at one of my camera entities under MQTT and have the following:
<img width="669" alt="Screenshot 2023-07-27 at 12 41 58 PM" src="https://github.com/tsightler/ring-mqtt/assets/57337383/98834039-fbfd-4e69-9a69-9191f2baae14">

Am I correct to assume if/when it is working properly, I would see under controls an entity for home mode that I could choose the state of Away, Home, etc?

If it should be there, I will go through the process of creating an issue.

**Reply by tsightler** [reply]
No, under devices you should see an entirely new alarm control panel device with the name in the format of "<location_name> Mode".  It won't be an entity of an existing device because you don't set modes for individual devices, you set modes for the location.
